src/components/DatepickerDay/index.js

```diff
@@ -21,6 +21,7 @@ const DatepickerDay = ({
   dateFormat,
   locale,
   withPortal,
+  isPink,
```

Comment:
Can't we take advantage of the `iconColor` I added before to implement this in an even more flexible way?

Reply:
Yes you r right, will be better to have something more flexible 
About that, Keanan wanted to discuss this afternoon during our frontend meeting about adding a rule to change color easily for all the flamingo components (because it's always annoying when we want a component with primary or secondary styles), as it's related I leave it as it's for now :)